src/resources/2025/LepreCondo.ts

```diff
@@ -0,0 +1,315 @@
+import { Effect, Item, visitUrl } from "kolmafia";
+import { directlyUse, have as have_ } from "../../lib.js";
+import { $effect, $item, $items } from "../../template-string.js";
+import { get } from "../../index.js";
+
+/**
+ * @returns Whether you `have` the Leprecondo
+ */
+export function have(): boolean {
+  return have_($item`Leprecondo`);
+}
+
+/**
+ * @returns How many more times you can rearrange today
+ */
+export function rearrangeUses(): number {
+  return get("_leprecondoRearrangements");
+}
+
+function parseFurniture(str: string): Furniture[] {
+  return str
+    .split(",")
+    .map(Number)
```

Comment:
no need for this to be two separate maps, you can do `.map((index) => itemsMap[Number(index)])`

Reply:
Also given that the elements of `itemsMap` aren't themselves `Furniture` by the typedef you lay down below, this won't work great